tests/functional/test_bundle_support.py

```diff
@@ -158,23 +161,46 @@ def test_run_parametrized_action_must_be_runned(sdk_client_fs: ADCMClient):
     ("provider", "on_fail", "was_dict"),
     ("provider", "on_fail", "was_list"),
     ("provider", "on_fail", "was_sequence"),
-    ("host", "on_success", "was_dict"),
-    ("host", "on_success", "was_list"),
-    ("host", "on_success", "was_sequence"),
-    ("host", "on_fail", "was_dict"),
-    ("host", "on_fail", "was_list"),
-    ("host", "on_fail", "was_sequence"),
+    ("cluster", "on_success", "was_dict_new_dsl"),
+    ("cluster", "on_success", "was_list_new_dsl"),
+    ("cluster", "on_success", "was_sequence_new_dsl"),
+    ("cluster", "on_fail", "was_dict_new_dsl"),
+    ("cluster", "on_fail", "was_list_new_dsl"),
+    ("cluster", "on_fail", "was_sequence_new_dsl"),
```

Comment:
how about services and components?

Reply:
I guess cluster is enough. Due to the common functions used. Maybe we can add provider

Reply:
But overall it is about testing yaml validator